### ISSUE TITLE
Output triggers the clippy double_parens warning

### ISSUE THREAD

**Issue by ndmitchell** [lalrpop/lalrpop#518]
When using an lalrpop file with my clippy configuration I get a warning about double_parens, because of code such as:

```
                Token::ByteLiteral(__tok0) if true => __Symbol::Variant1((__tok0)),
```

Would it be possible to remove the double brackets surrounding `__tok` and avoid this warning?

**Comment by Marwes** [comment]
Do you use 0.18.1? I fixed some instances like this in #506 .

**Comment by ndmitchell** [comment]
Apologies, I was indeed pinned to an old version. Many thanks!
